steps/xcuitestbrowserstack/1.0.3/step.yml

```diff
@@ -0,0 +1,80 @@
+title: Browserstack iOS UI Test
+summary: |
+  Execute XCUITests on Browserstack Platform with retry logic
+description: |
+  This step will help you to execute iOS UI Automation tests on Browserstack platform. Automatically manages device queue and retries failed tests and also reports test status.
+website: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+source_code_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+support_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack/issues
+published_at: 2019-08-31T10:57:32.089186+04:00
+source:
+  git: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack.git
+  commit: d3ae107234259f383e723c0aa84fd305818e7a38
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: browserstack.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- browserstack_app_url: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: IPA url of the app uploaded to Browserstack Server. You can use browserstack-upload
+      bitrise step to upload your ipa.
+    title: Browserstack ipa url
+- opts:
+    description: |
+      Sample file format:
+          SearchListingsTest/testXYZ=iPhone 6-11.2
+          SearchFilterTest/testABC=iPhone XS-12.1
+    is_expand: true
+    is_required: true
+    summary: Specifiy path to the file which contains your test and device details.
+      Use .txt file to specify your test name and device details separated by equal
+      to sign.
+    title: Test file path
+  tests_file_path: null
+- browserstack_xcuitest_url: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Build your XCUITests and publish it to Browserstack sever. Specify the
+      test_url path returned from browserstack in this variable.
+    title: Browserstack xcuitest url
+- browserstack_username: null
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Browserstack username
+- browserstack_password: null
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Browserstack access key
+- browserstack_local: false
```

Comment:
Please specify inputs as string and add a value_options to have a drop-down to choose from.
```  
- verbose_log: "true"
    opts:
      title: "Title"
      value_options:
        - "true"
        - "false"
```